lib/features/home_page/widgets/type.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex/utilities/colors.dart';
+import 'package:pokedex/utilities/extensions.dart';
+
+class Type extends StatelessWidget {
+  const Type({
+    this.typeName,
+    this.pokemonType,
+  });
+
+  final String? typeName;
+  final String? pokemonType;
```

Comment:
Could you change this so that it will accept a color instead of the `pokemonType`